src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -298,7 +300,7 @@ internal static Diagnostic Create(DiagnosticInfo info)
         /// <summary>
         /// Get the culture specific text of the message.
         /// </summary>
-        public abstract string GetMessage(IFormatProvider formatProvider = null);
+        public abstract string? GetMessage(IFormatProvider? formatProvider = null);
```

Comment:
>GetMessage [](start = 32, length = 10)

Does this ever return `null`?

Reply:
Apparently not after unwinding other returns

---
In reply to: [356889626](https://github.com/dotnet/roslyn/pull/40268#discussion_r356889626) [](ancestors = 356889626)